openshift/kustomize/tekton/base/pipelines/buildah-all.yaml

```diff
@@ -23,13 +23,20 @@ spec:
       description: >-
         The git revision reference to build.
       type: string
+      default: ""
 
     - name: CONTEXT
       description: >-
         The root path to the git repo.
       type: string
       default: tno
 
+    - name: COMPONENT
```

Comment:
At some point I'd like to to be able to handle a space separated list of components to build/deploy.